configs/xtensa-espressif_esp32s2_zephyr-elf.config

```diff
@@ -0,0 +1,66 @@
+CT_CONFIG_VERSION="3"
+CT_EXPERIMENTAL=y
+CT_LOCAL_TARBALLS_DIR="${CT_PREFIX:-${HOME}/x-tools}/sources"
+# CT_PREFIX_DIR_RO is not set
+# CT_LOG_PROGRESS_BAR is not set
+CT_TARGET_CFLAGS="-mlongcalls"
+CT_ARCH_XTENSA=y
+CT_XTENSA_CUSTOM=y
+CT_OVERLAY_NAME="espressif_esp32s2"
+CT_OVERLAY_LOCATION="./overlays"
+CT_TARGET_VENDOR="espressif_esp32s2_zephyr"
+CT_BINUTILS_SRC_CUSTOM=y
+CT_BINUTILS_CUSTOM_LOCATION="${GITHUB_WORKSPACE}/binutils"
+CT_NEWLIB_SRC_CUSTOM=y
+CT_NEWLIB_CUSTOM_LOCATION="${GITHUB_WORKSPACE}/newlib"
+CT_LIBC_NEWLIB_TARGET_CFLAGS="-O2"
+# CT_LIBC_NEWLIB_ENABLE_TARGET_OPTSPACE is not set
+CT_LIBC_NEWLIB_IO_C99FMT=y
+CT_LIBC_NEWLIB_IO_LL=y
+CT_LIBC_NEWLIB_IO_FLOAT=y
+# CT_LIBC_NEWLIB_FSEEK_OPTIMIZATION is not set
+CT_LIBC_NEWLIB_DISABLE_SUPPLIED_SYSCALLS=y
+CT_LIBC_NEWLIB_GLOBAL_ATEXIT=y
+CT_LIBC_NEWLIB_LITE_EXIT=y
+CT_LIBC_NEWLIB_MULTITHREAD=y
+CT_LIBC_NEWLIB_RETARGETABLE_LOCKING=y
+# CT_LIBC_NEWLIB_WIDE_ORIENT is not set
+# CT_LIBC_NEWLIB_NANO_MALLOC is not set
+# CT_LIBC_NEWLIB_NANO_FORMATTED_IO is not set
+CT_LIBC_NEWLIB_EXTRA_SECTIONS=y
+CT_GCC_SRC_CUSTOM=y
+CT_GCC_CUSTOM_LOCATION="${GITHUB_WORKSPACE}/gcc"
+CT_CC_GCC_EXTRA_CONFIG_ARRAY="--with-gnu-ld --with-gnu-as --enable-initfini-array"
+CT_CC_LANG_CXX=y
+CT_DEBUG_GDB=y
+CT_GDB_SRC_CUSTOM=y
+CT_GDB_CUSTOM_LOCATION="${GITHUB_WORKSPACE}/gdb"
+CT_GDB_CROSS_PYTHON_BINARY="python3.8"
+CT_GDB_CROSS_BUILD_NO_PYTHON=y
```

Comment:
Please remove this too as per https://github.com/zephyrproject-rtos/sdk-ng/commit/5abd8f7db6f53d7ba994a93d82d7fde783a3c691.